src/utils/requests.ts

```diff
@@ -0,0 +1,22 @@
+import type { JSONType } from "@/types/json";
+
+const SERIALIZED_TYPES = ["object", "array"];
+
+export const createFormData = (obj: Record<string | number, JSONType>) => {
+  const formData = new FormData();
+  Object.entries(obj).forEach(([k, v]) => {
+    if (v === undefined || v === null) return;
```

Comment:
What is the purpose of this early return?
I fear this could lead to strange behaviour when launching a workflow with optional parameters, as it could result in an empty FormData if the first value is `undefined | null`.

Reply:
form data entries must be strings - if the form data key isn't specified, that's equivalent to a null value in practice.